### PR TITLE
fixed timeout for pbs_hook_alarm_large_multinode_job.py and moved to …

### PR DESCRIPTION
**Pull request by anamikau**
…resilience folder

updated pbs_soft_walltime race condition in test_soft_held

#### Bug/feature Description
* *pbs_hook_alarm_large_multi-node_job.py was timing out*
* *test_soft_held failed while checking estimated soft_walltime as the value was increased*

#### Affected Platform(s)
* *Platform ALL*

#### Cause / Analysis / Design
* *Bugs: timeout was added to wrong place for pbs_hook_alarm_large_multi-node_job.py. And test_soft_held: total sleep is 37s out of which 10s when the job is held. minus 10s, job is sleeping for 27s adding race condition while checking the value*

#### Solution Description
* *pbs_hook_alarm_large_multi-node_job.py: moved timeout at each test and also increased the debugging for server and mom*
* *test_soft_held: Checking for 2 values for softwalltime*
[test_soft_held.txt](https://github.com/PBSPro/pbspro/files/1896724/test_soft_held.txt)
[TestPbsHookAlarm.txt](https://github.com/PBSPro/pbspro/files/1896725/TestPbsHookAlarm.txt)

#### Checklist:

- [x] I have joined the **[pbspro community forum](http://community.pbspro.org/)**.
- [x] My pull request contains a **single, signed** commit. See **[setting up gpg signature](https://pbspro.atlassian.net/wiki/display/DG/Signing+Your+Git+Commits)**.
- [ ] My code follows the **[coding style](https://pbspro.atlassian.net/wiki/display/DG/Coding+Standards)** of this project.
- [ ] My change requires project documentation. See **[required documentation checklist](https://pbspro.atlassian.net/wiki/display/DG/Checklist+for+Developing+Features+and+Bug+Fixes)** for details.
   - [ ] I have added documentation in the **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)**.
- [ ] I have added new **PTL test(s) to my commit**. (See **[using PTL for testing](https://pbspro.atlassian.net/wiki/display/DG/Using+PTL+for+Testing)**) *(or)*
   - [ ] I have added  **manual test(s) to this pull request and explained why PTL is not appropriate** for this case.
- [ ] All new and existing automated tests have passed. (See **[running automated PTL tests](https://pbspro.atlassian.net/wiki/display/DG/PTL+Quick+Start+Guide)**).
- [ ] I have attached **test logs to this pull request** as evidence of testing/verification.

__***For further information please visit the [Developer Guide Home](https://pbspro.atlassian.net/wiki/display/DG/Developer+Guide+Home).***__

### CODE REVIEW

File: test/tests/functional/pbs_soft_walltime.py [hunk 1]
```diff
@@ -889,8 +889,8 @@ def test_soft_held(self):
             " since the walltime was reset to 0")
         time.sleep(17)
         self.server.expect(JOB, {'estimated.soft_walltime':
-                                 (MATCH_RE, '00:00:21|21')}, id=jid,
-                           max_attempts=5)
+                                 (MATCH_RE, '00:00:21|21|00:00:28|28')},
+                           id=jid)
```
Comment (bhroam) [inline test/tests/functional/pbs_soft_walltime.py]: I've recently learned that it's the server that converts a time based resource into HH:MM:SS.  You don't actually have to do the MATCH_RE.  Regardless of cli/api mode, it the resource will be in that format.  There's no harm to what you have done.  It is consistent with the rest of the test.

Comment (anamikau) [reply test/tests/functional/pbs_soft_walltime.py]: Thanks for the sharing Bhroam. Like you said I would leave it like that to be consistent with rest of the test.

**Comment by bhroam** [comment]
Please squash and rebase for the merge.

**Comment by anamikau** [comment]
All checks passed.
@bhroam rebased and squashed. Please do the needful.

**Comment by bhroam** [comment]
I merged in the PR

**Pull request merged**